nilearn/_utils/estimator_checks.py

```diff
@@ -262,7 +262,7 @@ def return_expected_failed_checks(
         "check_estimators_fit_returns_self": (
             "replaced by check_fit_returns_self"
         ),
-        "check_fit_check_is_fitted": ("replaced by check_masker_fitted"),
+        "check_fit_check_is_fitted": ("replaced by check_fit_check_is_fitted"),
```

Comment:
This looks weird. What is the point ?

Reply:
ha yeah this is because I have the named the check in nilearn the same as that in sklearn, let me change that